subprojects/emcee/queue-api/src/main/kotlin/com/avito/emcee/queue/TestTimeoutConfiguration.kt

```diff
@@ -0,0 +1,6 @@
+package com.avito.emcee.queue
+
+public data class TestTimeoutConfiguration(
+    val singleTestMaximumDuration: Int,
+    val testRunnerMaximumSilenceDuration: Int
```

Comment:
```suggestion
    val testRunnerMaximumSilenceDurationSec/Ms/...: Int
```
Let's clarify units if we can't use better types here and convert them by moshi.

Reply:
This part will be changed